jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/WatchService.java

```diff
@@ -417,6 +431,7 @@ private WatchMode getWatchMode(ImageConfiguration imageConfig) {
         private String showLogs;
         private Date buildTimestamp;
         private String containerNamePattern;
+        private Predicate<String> postGoalTask;
```

Comment:
I'm not sure if this configuration renders the `private String watchPostGoal;` (L417) field useless.
Could you double check?
In case it's no longer used, we need to either rename the new one, or update docs accordingly.

Reply:
hmm, Ok, I see the first is used by the second. Something seems a little bit bizarre here. Checking...

Reply:
See suggestion: https://github.com/eclipse/jkube/pull/402/files#r495938227